templates/index.html

```diff
@@ -227,6 +222,11 @@ <h3 class="section-head">Mental State Graph</h3>
                     success: function (data) {
                         data = data[0];
                         console.log(data);
+                        let date = data.timestamp;
+                        let emotion = data.emotion;
```

Comment:
Since you've created `date`, `emotion`, `confidence` variables, do you mind editing parts of my code that directly refer to `data.<field>` to use these variables instead?